### PR TITLE
Upgrade graphql-php to v0.13.6

### PR DESCRIPTION
**Pull request by pmclain**
### Description (*)
Follows upgrade path outlined in https://github.com/webonyx/graphql-php/blob/master/UPGRADE.md

### Fixed Issues (if relevant)
1. magento/graphql-ce#777

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by pmclain** [comment]
@magento run Functional Tests build

File: lib/internal/Magento/Framework/GraphQl/Schema/Type/ScalarTypes.php [hunk 1]
```diff
@@ -13,23 +13,27 @@
...
+     * Get instance of scalar type
+     *
      * @param string $typeName
      * @return \GraphQL\Type\Definition\ScalarType|\GraphQL\Type\Definition\Type
      * @throws \LogicException
      */
     public function getScalarTypeInstance(string $typeName) : \GraphQL\Type\Definition\Type
     {
-        $internalTypes = \GraphQL\Type\Definition\Type::getInternalTypes();
+        $internalTypes = \GraphQL\Type\Definition\Type::getStandardTypes();
```
Comment (paliarush) [inline lib/internal/Magento/Framework/GraphQl/Schema/Type/ScalarTypes.php]: Variable should be renamed accordingly.
Is "getInternalTypes" deprecated in the new version of webonyx?

**Review by paliarush — APPROVED** [review]
Library upgrade looks good, please address  one questions and comment regarding renaming variables.

File: lib/internal/Magento/Framework/GraphQl/Schema/Type/ScalarTypes.php (hunk 1 above)
Comment (pmclain) [reply lib/internal/Magento/Framework/GraphQl/Schema/Type/ScalarTypes.php]: I'll update. Yes, it was deprecated https://github.com/webonyx/graphql-php/blob/e804ccf73add32be937d392abb8b359243817d57/src/Type/Definition/Type.php#L189-L199

**Comment by pmclain** [comment]
@magento run Functional Tests build

**Comment by pmclain** [comment]
@magento run Functional Tests build

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5734](https://jira.corp.magento.com/browse/ENGCOM-5734) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by lenaorobei** [comment]
@pmclain could you please pull latest `2.3-develop` and resolve `lock` conflicts? Thanks.

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5734](https://jira.corp.magento.com/browse/ENGCOM-5734) has been created to process this Pull Request

**Pull request merged**